test/integration/minimal/controls/minimal.rb

```diff
@@ -73,4 +75,11 @@
       expect(liens).to be_empty
     end
   end
+
+  describe "group_email" do
+    it "should be empty if user_group is empty" do
+      expect(group_email).to be_empty
+      expect(group_name).to be_empty
```

Comment:
We should only have one assertion per example to simplify debugging when failures occur. Please move this to a separate `it` example.

Reply:
Thank you for your review. Done.